infer/src/absint/ProcCfg.ml

```diff
@@ -201,6 +201,98 @@ module Normal = struct
   let wto = Procdesc.get_wto
 end
 
+(** Forward .NET CFG with exceptional control-flow *)
+module DOTNETExceptional = struct
```

Comment:
It doesn't seem to me that this is specific to dotnet, is there a better name that would capture which exceptional edges are followed? `ExceptionalThrowOnly`? (see below)

Reply:
Done. Thanks for suggestion.